### DISCUSSION TITLE
Help or work for $... Simply aplication

### DISCUSSION THREAD

**Discussion by zimart** [Budibase/budibase#14300] [Help]
Hello, I discovered Budibase recently, I'm not very good at it. I need help, or possibly work for $... :)
I have a problem with automatic completion of form field values, or immediately a table in the database - using a Query.
If someone would like to help, or I can pay for the work - what is the right way ? . Building tables and exporting the application, or access to the Budibase account?
The application should be very simple - saving customer offers - the offer contains a list of products with automatic retrieve of the product price from another product table based on a field generated when filling in form values ​​(SKU).
To sum up - we create a customer, add an offer to him and add products in the offer (I also want an in-line editable version)
The "basket" table contains e.g. type, width, height, color.
Based on the filled fields, I generate the SKU code using JavaScript (I will do that), and at this point I need to automatically download the price from another table (based on the generated SKU - not using a relationship).

Please help me, how to do it (retrieve the price from another table). Database - Mysql. Maybe someone will provide me with an example application with the database - for learning purposes :)
